### ISSUE TITLE
Error msg: 'div' html tag with 'markdown' attribute

### ISSUE THREAD

**Issue by theandychung** [bordaigorl/sublime-evernote#207]
I clipped a webpage to evernote and open it in sublime text 3 using this package. I found this package is not able to update my note back to evernote if my note has more than 5 layers of div. The error msg i get is the following:
> Evernote Complained:
> The contents of the note are not valid.
> The note contains a 'div' HTML tag with a 
> 'markdown' attribute; this is not allowed in a note.
> 
> Retry?

Here is a sample code I made:
```html
<div markdown="1" style="-evernote-webclip: true;">
<div markdown="1" style="font-size: 16px; display: inline-block;">
<div markdown="1" style="font-size: 16px; display: inline-block;">
<div markdown="1" style="font-size: 16px; display: inline-block;">
<div markdown="1" style="font-size: 16px; display: inline-block;">
xxxx

</div>
</div>
</div>
</div>
</div>
```

I noticed that the similar extension in vscode, evermonkey, doesn't add  markdown="1" when pulling the notes from evernote server, and it doesn't have any problem uploading the note back to evernote. I wonder if disabling automatically adding these markdown attributes could solve this issue.

Thank you for making this wonderful package for us.

**Comment by nonbeing** [comment]
I'm getting the same error. 

And yes, editing and publishing the same note in VS Code's "evermonkey" extension works fine without any errors. This used to work fine a while ago (more than a year ago when I last used `sublime-evernote`, so I guess the API or something's changed since then and this extension needs to be updated.

**Comment by bordaigorl** [comment]
I looked into this.
The problem is twofold:
1. the Evernote format keeps changing without documentation and inconsistently across clients.
2. the markdown converter we use in this plugin is crap.

Fixing 2 seems trivial but is not as the conversion needs to be heavily customised to be compatible with Evernote's restrictions on the notes content.

I am going to push a patch that should help with these cases. I need your help with testing. And there's no guarantee it works in all cases and that it will work in the future.

**Comment by bordaigorl** [comment]
@theandychung @nonbeing I would be very grateful if you could install the new-codeblocks branch and test it with your workflow to see if the problems are solved.

**Comment by theandychung** [comment]
I can't upgrade the package through package control, do I need to upgrade it manually?

**Comment by bordaigorl** [comment]
@theandychung yes you need to do this manually (I'm not updating the master version without testing 😉 )
One way would be to replace the package in sublimetext by cloning the `new-codeblocks` branch in the Packages/Evernote directory.
You can put the old version back when you are done.

**Comment by theandychung** [comment]
It's the same. The website I'm having problem is [this one](https://www.lockheedmartinjobs.com/job/sunnyvale/guidance-navigation-and-control-engineer/694/9504410)

**Comment by bordaigorl** [comment]
@theandychung could you please give some more details?
I need:

1. which platform you are using (os, browser, evernote client)
2. what actions you take: what do you do with the webclipper? Save selection? The whole page?
3. what you expect to happen when you open a webclip from sublime text
4. what you expect to happen when you edit and save a webclip from sublime text
5. what happens instead.

Thanks

**Comment by bordaigorl** [comment]
I did some tests on the page you linked.
It turns out a different problem was kicking in: the webclipper is inlining styles, including fonts and delimiting font names with `&quot;`. That caused the plugin to first turn these into `"` upon conversion and then getting confused because of non matching `"` in the attributes of the divs.
I just pushed a patch in `new-codeblocks`, try pull and test again.

**Comment by theandychung** [comment]
Sorry, I should have made it more clear. I used webclipper to clip a [section of the page](https://imgur.com/L7yxffQ), then open it in the sublime text 3, then trying to update the note. The new patch didn't work for me.

**Comment by bordaigorl** [comment]
@theandychung Can you explain what went wrong?

**Comment by theandychung** [comment]
The error msg I mentioned at the beginning pops out whenever I want to update the note back to evernote (even if I didn't edit anything). I wonder if this is an unexpected behavior.
